grabdish/inventory-micronaut/src/main/java/io/micronaut/data/examples/InventoryServiceOrderEventConsumer.java

```diff
@@ -39,12 +42,74 @@ public InventoryServiceOrderEventConsumer(DataSource atpInventoryPDB) {
     public void run() {
         System.out.println("Receive messages... ");
         try {
-            listenForOrderEvents();
+            listenForOrderEventsNew();
         } catch (Exception e) {
             e.printStackTrace();
         }
     }
 
+    public void listenForOrderEventsNew() throws Exception {
```

Comment:
could we call all these method changes "Queue" and "Topic" instead of "Old" and "New" to better describe?

Reply:
Good idea.  I will make that change.